kubernetes/linux/setup.sh

```diff
@@ -62,7 +68,8 @@ sudo tdnf install fluent-bit-2.0.9 -y
 echo "$(fluent-bit --version)" >> packages_version.txt
 
 # install fluentd using the mariner package
-sudo tdnf install rubygem-fluentd-1.14.6 -y
+# sudo tdnf install rubygem-fluentd-1.14.6 -y
+gem install fluentd -v "1.14.6" --no-document
```

Comment:
what's preventing us to install gem from tdnf?

Reply:
With tdnf install of fluentd also uses the same compiler flags and hence the memory is high for that as well...

Reply:
Ok, got it, thanks.